### DISCUSSION TITLE
What enchantment goes to what item

### DISCUSSION THREAD

**Discussion by itoaste** [Geolykt/EnchantmentsPlus#128] [Q&A]
In the wiki could you state what enchantment goes to what item? With most of them it's self explanatory, but with some i have no clue.

**Comment by Geolykt** [comment]
You can deduct the default values from [https://github.com/Geolykt/EnchantmentsPlus/blob/4xx/resources/defaultconfig.yml#L38](https://github.com/Geolykt/EnchantmentsPlus/blob/4xx/resources/defaultconfig.yml#L38). Furthermore you can do `/ench list` in game to see what enchantments can be applied on your item
